src/qt/locale/bitcoin_fr.ts

```diff
@@ -5,35 +5,35 @@
     <name>AddPeerDialog</name>
     <message>
         <source>Add Peer</source>
-        <translation type="unfinished"></translation>
+        <translation>Ajouter un pair</translation>
     </message>
     <message>
         <source>Enter the peer details below.</source>
-        <translation type="unfinished"></translation>
+        <translation>Entrez les détails du pair ci-dessous.</translation>
     </message>
     <message>
         <source>Be careful! Do not blindly trust anyone that tells you to add their node.</source>
-        <translation type="unfinished"></translation>
+        <translation>Soyez prudent ! Ne faites pas aveuglément confiance à une personne qui vous demande d&apos;ajouter son nœud.</translation>
```

Comment:
Je changerais pour "Ne faites pas confiance aveuglément à une personne..." semble plus naturel.

Reply:
This is a bit of a blur compared to the original English instruction.